src/main/java/uk/gov/register/filters/ContentSecurityPolicyFilter.java

```diff
@@ -12,6 +12,6 @@
 public class ContentSecurityPolicyFilter implements ContainerResponseFilter {
     @Override
     public void filter(ContainerRequestContext requestContext, ContainerResponseContext responseContext) throws IOException {
-        responseContext.getHeaders().add(HttpHeaders.CONTENT_SECURITY_POLICY, "default-src 'self' www.google-analytics.com");
+        responseContext.getHeaders().add(HttpHeaders.CONTENT_SECURITY_POLICY, "default-src 'self' www.google-analytics.com; font-src data:;");
```

Comment:
From verify.gov.uk - "I agree data: would be a bad idea on script-src but I don’t see an issue using it for font-src. Given the GOV.UK font is served that way currently so unless that changes there is no real alternative."